source/includes/_wpc.md

```diff
@@ -62,6 +62,7 @@ curl -H "Authorization: Bearer OAUTH_ACCESS_TOKEN" \
 "https://api.localiqservices.com/client_reports/wpc/TEST_1?campaign_status[]=running&campaign_status[]=stopped&start_date=2016-10-01&end_date=2016-12-31"
 ```
 
+CHANGE THIS RESPONSE
```

Comment:
This needs to be completed.  I should have called out that this was a placeholder.  I figured it would be easier to just call the API and copy the response after development was complete.